features/rageshake/impl/src/main/kotlin/io/element/android/features/rageshake/impl/reporter/DefaultBugReporter.kt

```diff
@@ -156,6 +160,19 @@ class DefaultBugReporter @Inject constructor(
                         .addFormDataPart("user_id", userId)
                         .addFormDataPart("can_contact", canContact.toString())
                         .addFormDataPart("device_id", deviceId)
+                        .apply {
+                            userId.takeIf { MatrixPatterns.isUserId(it) }?.let {
+                                SessionId(it)
```

Comment:
`sessionData?.userId` is guaranteed to be a valid userId, so to simplify, I would change line 148 from 
`val userId = sessionData?.userId ?: "undefined"`
to
`val userId = sessionData?.userId?.let { UserId(it) }` 

and use `?: "undefined"` at line 148.

Please be aware that the value can still be null (user can send bug report before having a session.

Reply:
Updated